lib/connect-session-sequelize.js

```diff
@@ -68,16 +74,32 @@ module.exports = function SequelizeSessionInit (Store) {
           }
           debug('FOUND %s with data %s', session.sid, session.data)
 
+          if (self.options.secret) {
+            self.options.kruptein.get(self.options.secret, session.data, function (err, plaintext) {
+              if (err) return null
+
+              session.data = JSON.parse(plaintext)
+            })
```

Comment:
if kruptein is async then setting session.data here won't work.

Reply:
It is async and if you enable `debug` in your module you can see the cipher text object on both `.get()` and `.set()`.